internal/pkg/n8n-client-go/client_integration_test.go

```diff
@@ -40,3 +41,119 @@ func TestIntegrationGetWorkflows(t *testing.T) {
 		t.Errorf("expected 0 workflows, got %d", len(workflows.Data))
 	}
 }
+
+func TestIntegrationCreateWorkflow(t *testing.T) {
+	// Start the n8n container for testing
+	container, url, err := helpers.CreateTestContainer()
+	require.NoError(t, err)
+
+	defer func() {
+		if err := helpers.TerminateContainer(context.Background(), container); err != nil {
+			fmt.Println("Error when terminating container:", err)
+		}
+	}()
```

Comment:
Can we replace the inline function with a reusable function that can be called repeatably in many integration tests?

This should save some lines of code and speed-up updates in case we need to change the error message.

Example below:
```suggestion
        defer terminateContainer(context.Background(), container)

        func terminateContainer(ctx context.Context, container someContainerType) {
	        if err := helpers.TerminateContainer(ctx, container); err != nil {
		        fmt.Println("Error when terminating container:", err)
	        }
        }
```